user-access-commons/src/main/java/no/unit/nva/useraccessmanagement/model/ViewingScope.java

```diff
@@ -83,26 +82,39 @@ public boolean equals(Object o) {
             return false;
         }
         ViewingScope that = (ViewingScope) o;
-        return isRecursive() == that.isRecursive()
```

Comment:
removed recursion because it is not used now, and it made refactoring much harder.